hasher-matcher-actioner/requirements.txt

```diff
@@ -1,5 +1 @@
-boto3
-boto3-stubs[essential]==1.17.14.0
-threatexchange[faiss,pdq_hasher]>=0.0.15
-bottle
-apig_wsgi
+.
```

Comment:
I believe this will also install hmalib locally, which may not be good, but I can no longer test the complete install path due to mysterious windows compile failures.

Reply:
Why not delete this file?

Reply:
Delaying replacing some references to it in actions